tests/Microsoft.Identity.Test.Unit.net45/ExceptionTests/MsalExceptionTests.cs

```diff
@@ -3,15 +3,268 @@
 
 using System;
 using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
 using Microsoft.Identity.Client;
 using Microsoft.Identity.Client.Http;
+using Microsoft.Identity.Client.Internal;
+using Microsoft.Identity.Client.OAuth2;
+using Microsoft.Identity.Client.Utils;
+using Microsoft.Identity.Test.Common.Core.Helpers;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
 namespace Microsoft.Identity.Test.Unit.ExceptionTests
 {
     [TestClass]
     public class MsalExceptionTests
     {
+        private const string ExCode = "exCode";
```

Comment:
>exCode [](start = 39, length = 6)

can you put some of these in the const file for tests?

Reply:
These constants are private, i.e. they are only used in this class. I can't see other tests that would benefit from them yet.